packages/labs/analyzer/src/lib/errors.ts

```diff
@@ -5,27 +5,52 @@
  */
 
 import ts from 'typescript';
+import {DiagnosticCode} from './diagnostic-code.js';
+import {EOL} from 'os';
```

Comment:
I'd like to think forward to running the analyzer in non-Node environments... how does depending on Node packages here effect that?

And does TypeScript have a base/default FormatDiagnosticsHost that we can use?